tests/ast/cairo-keywords-for-names.ast.result

```diff
@@ -1,56 +1,10 @@
-Block:
-	FunctionDefinition:
-		Name: func__warp_mangled
-		Parameters:
-			TypedName:
-				felt__warp_mangled:Uint256
-			TypedName:
-				end__warp_mangled:Uint256
-		Return Variables:
-			TypedName:
-				jmp__warp_mangled:Uint256
-		Body:
-			Block:
-				VariableDeclaration:
-					Variables:
-						TypedName:
-							_1_7:Uint256
-					Value:
-						FunctionCall:
-							Identifier:iszero
-							Identifier:felt__warp_mangled
-				VariableDeclaration:
-					Variables:
-						TypedName:
-							_2_8:Uint256
-					Value:
-						FunctionCall:
-							Identifier:iszero
-							Identifier:_1_7
-				ExpressionStatement:
-					FunctionCall:
-						Identifier:require_helper
-						Identifier:_2_8
-				VariableDeclaration:
-					Variables:
-						TypedName:
-							_3_9:Uint256
-					Value:
-						Literal:100
-				VariableDeclaration:
-					Variables:
-						TypedName:
-							ret__warp_mangled:Uint256
-					Value:
-						FunctionCall:
-							Identifier:checked_add_uint256
-							Identifier:end__warp_mangled
-							Identifier:felt__warp_mangled
-				Assignment:
-					Variables:
-						Identifier:jmp__warp_mangled
-					Value:
-						FunctionCall:
-							Identifier:gt
-							Identifier:ret__warp_mangled
-							Identifier:_3_9
\ No newline at end of file
+{
+	function func__warp_mangled(felt__warp_mangled : Uint256, end__warp_mangled : Uint256) jmp__warp_mangled : Uint256 {
```

Comment:
Isn't there supposed to be a `->` before the return variable?

Reply:
Oh yes, that's true! I'll add it.